src/routes/(app)/play/[[id]]/+page.svelte

```diff
@@ -5,25 +5,36 @@
     import Board from '$lib/game/Board.svelte';
     import { GameStatus, PlayerColor, Scoring } from '$lib/game/enums';
     import { Game } from '$lib/game/game';
+    import { browser } from '$app/environment';
 
     export let data: PageData;
 
     let game: Game | undefined = undefined;
 
     $: status = game?.status;
     $: history = game?.history;
-    $: currentPlayer = game?.currentPlayer;
 
-    $: if (data.game) {
+    $: if (data.game && browser) {
         game = Game.init(data.game);
     }
 
     function onAcceptDeadStonesClick(): void {
         game?.finishGame();
     }
 
-    function onPassClick(): void {
-        game?.pass();
+    async function onPassClick(): Promise<void> {
+        if (!game) return;
+
+        game.pass();
+
+        let formData = new FormData();
```

Comment:
convert to regular obj. Also, consider adding a type for this data transfer object, which can/will be used in the frontend and backend.